core/src/utils/time.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { time } from "./time";
+
+beforeAll(() => {
+  jest.useFakeTimers();
+});
+
+test("time - 기본적으로 getTime() 값을 반환하며, 이전 호출과 현재 호출 사이에 시간이 동일한 경우 중복을 방지한다.", () => {
+  const expectedTime1 = new Date("2023-08-06").getTime(); // 1691280000000
+  const expectedTime2 = new Date("2023-08-07").getTime(); // 1691366400000
+
+  jest.setSystemTime(expectedTime1);
+
+  expect(time()).toBe(expectedTime1);
+  expect(time()).toBe((expectedTime1 * 1000 + 1) / 1000);
+  expect(time()).toBe((expectedTime1 * 1000 + 2) / 1000);
+  expect(time()).toBe((expectedTime1 * 1000 + 3) / 1000);
+
+  jest.setSystemTime(expectedTime2);
+
+  expect(time()).toBe(expectedTime2);
+  expect(time()).toBe((expectedTime2 * 1000 + 1) / 1000);
+  expect(time()).toBe((expectedTime2 * 1000 + 2) / 1000);
+  expect(time()).toBe((expectedTime2 * 1000 + 3) / 1000);
+});
```

Comment:
```suggestion
test("time - 동일한 시간에 여러번 호출될 경우 중복을 방지한다.", () => {
  // given
  jest.setSystemTime(new Date("2023-08-06").getTime());

  // when
  const time1 = time();
  const time2 = time();
  
  // then
  expect(time1).not.toEqual(time2)
});
```

issue: 유닛 테스트를 작성할 때는 구현 **세부**보다 구현 **의도**를 테스트 하는 것을 권장합니다.

`time` 함수의 구현 **의도**는 `동일한 시간에 여러번 호출 될 경우에도 uniqueness가 보장되는 getTime()` 입니다. 구현 **세부**는 작성해주신 테스트와 같이 `동일한 시간에 여러번 호출되는 경우, 1/1000씩 더해진 값을 반환한다`에 해당하지만, 이는 `time`함수에 이미 서술되어 있는 내용으로, 테스트에서는 이에 대한 코드를 작성하지 않는 것을 권장합니다.

코드가 변경될 시에도 동일한 input에 대해 동일한 output이 보장되는 것에 대한 테스트가 필요하다면, [Inline Snapshot](https://jestjs.io/docs/snapshot-testing#inline-snapshots) 테스트를 작성하는 것을 권장합니다.

Reply:
@malangcat 
comment: 이해했습니다🙇🏻‍♂️ 매번 디테일 한 리뷰에 감사드립니다. 테스트 코드는 수정하겠습니다